lib/compile.py

```diff
@@ -30,8 +30,12 @@ def add_dialect(table_dict, table):
         # dialect related key-value pairs
         # copied as is
         if key == "nulls":
-            table_dict[
-                'cleanup'] = "Cleanup(correct_invalid_value, nulls=" + str(val) + ")"
+            try: 
+                table_dict[
+                    'cleanup'] = "Cleanup(correct_invalid_value, nulls=" + int(val) + ")"
+            except: 
+                table_dict[
+                    'cleanup'] = "Cleanup(correct_invalid_value, nulls=\"" + str(val) + "\")"
 
         elif key == "delimiter":
```

Comment:
I looked into this and it seems like this isn't an easy fix, either test will have to be changed, or some other parts of how null values are handled, so for now I've set compile.py back to its original form, and I'll create an issue for this.